compiler/src/iree/compiler/Codegen/Common/EncodingUtils.h

```diff
@@ -9,16 +9,47 @@
 
 #include "iree/compiler/Dialect/Encoding/IR/EncodingOps.h"
 #include "iree/compiler/Dialect/HAL/IR/HALTypes.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
```

Comment:
I think this is not needed.

Reply:
That one was actually needed as this header needs `tensor::PackOp` etc.